docs/excel/excel-add-ins-ranges.md

```diff
@@ -526,21 +526,47 @@ Excel.run(function (context) {
 }).catch(errorHandlerFunction);
 ```
 
-**Data in range before number format is set**
+#### Data in range before number format is set
 
 ![Data in Excel before format is set](../images/excel-ranges-format-font-and-fill.png)
 
-**Data in range after number format is set**
+#### Data in range after number format is set
 
 ![Data in Excel after format is set](../images/excel-ranges-format-numbers.png)
 
+## Read or write to an unbounded range
+
+### Read an unbounded range
+
+An unbounded range address is a range address that specifies either entire column(s) or entire row(s). For example:
```

Comment:
Should avoid (s) to semi-pluralize as it isn't read well by some screen readers.
```suggestion
An unbounded range address is a range address that specifies either entire columns or entire rows. For example:
```